src/creatwth.cpp

```diff
@@ -7,7 +7,7 @@
 //  Copyright (c) Microsoft Corporation.  All rights reserved.
 //
 
-// #define DETOUR_DEBUG 1
+// #define DETOUR_DEBUG 1GetModuleHandle(TEXT("kernel32.dll")
```

Comment:
copy paste mistake?

Reply:
yes, 😁,  it fix now, and not important because in comment.